example/src/main/resources/ConfigurationHelloBeautifulWorld.xml

```diff
@@ -3,11 +3,11 @@ Download ibisdoc.xsd from the Ibis console (via /rest/webservices). You can
 refer to it directly in the xsi:noNamespaceSchemaLocation attribute but it is
 better to refer to it using Eclipse, Window, Preferences, XML, XML Catalog,
 Add..., Location: path/to/ibisdoc.xsd, Key:
```

Comment:
I don't know if it's the place to rename this to "frankdoc.xsd", but just above perhaps change "Ibis console" to "Frank console"

Reply:
De gehele tekst klopt volgens mij niet, ik had een redelijk simpele find en replace gedaan, zonder inhoudelijk naar de bestanden te kijken. Ik zal ik zoeken op ibis...

Reply:
Oei, daar werd ik niet vrolijk van..

Reply:
Dat is waarschijnlijk allemaal al heel lang niet aangepast. Allemaal examples, niet iets waar iemand (van ons) ooit naar omkijkt natuurlijk!

Reply:
PS: Ik snap dat jij een simpele search/replace deed maar de diff liet mij toevallig een glimps zien van wat extra dingen.

Als jij nu na een search op "Ibis" zegt niet vrolijk ervan te worden, dan zie ik nog een paar van dit soort PRs in mijn nabije toekomst... 😁